src/pages/cluster-management/clusterInfo.html

```diff
@@ -18,139 +20,92 @@ <h1>
      size="75">
 </div>
 
-<div ng-hide="loader" class="clearfix mb-2">
-    <button type="button" class="btn btn-secondary delete-cluster-btn mr-2" ng-if="clusterConfiguration"
+<div ng-hide="loader" class="clearfix mb-2" ng-if="clusterConfiguration">
+    <button type="button" class="btn btn-secondary delete-cluster-btn mr-2"
             ng-click="showDeleteDialog()"
-            tooltip="{{'cluster_management.buttons.delete_cluster_tooltip' | translate}}" tooltip-placement="bottom">
+            tooltip="{{'cluster_management.buttons.delete_cluster_tooltip' | translate}}"
+            tooltip-placement="bottom">
         <span class="icon-trash"></span> {{'cluster_management.buttons.delete_cluster' | translate}}
     </button>
+    <button class="btn btn-link pull-right"
+            ng-click="toggleSidePanel()"
+            tooltip="{{'cluster_management.buttons.show_cluster_settings_tooltip' | translate}}"
+            tooltip-placement="left"><span class="icon-settings settings-icon small"></span></button>
 </div>
 
-<div ng-if="clusterConfiguration" class="row">
-    <div class="col-lg-6">
-        <div class="card cluster-information-card">
-            <div class="card-block">
-                <div class="clearfix">
-                    <h3 class="mb-0">
-                        <span class="collapsible-heading" data-toggle="collapse" data-target="#cluster-status"
-                              aria-expanded="false" aria-controls="cluster-status">
-                            <em class="icon-caret-down text-muted"></em>
-                              {{'cluster_management.cluster_status.label' | translate}}
-                        </span>
-                    </h3>
+<div ng-if="!loader" class="nodes">
+    <div cluster-graphical-view></div>
+</div>
+
+<pageslide
+    ps-class="rdf-info-side-panel"
+    ps-open="shouldShowClusterSettingsPanel"
+    onopen="onopen"
+    onclose="onclose"
+    ps-side="right"
+    ps-custom-height="calc(100vh - 55px)"
+    ps-size="25vw">
+    <div class="rdf-side-panel-content break-word-alt p-1 pt-2">
+       <span class="toggle-sidepanel" title="{{'visual.collapse.sidepanel' | translate}}" ng-click="toggleSidePanel()">
+            <em ng-class="shouldShowClusterSettingsPanel ? 'icon-caret-right' : 'icon-caret-left'"></em>
+        </span>
+        <button class="close mb-1 ml-1" ng-click="toggleSidePanel()"></button>
+        <h3 class="hovered-parent">
+            {{'cluster_management.cluster_configuration.label' | translate}}
+        </h3>
+        <h4>{{'cluster_management.cluster_configuration.configuration_label' | translate}}</h4>
+        <div class="rdf-l2ist">
```

Comment:
`rdf-l2ist`?

Reply:
This has to be removed. Removed the whole `div` wrapper.